test/test_stochastic.py

```diff
@@ -184,8 +185,34 @@ def test_dual_sgd_sinkhorn():
 
     # check constratints
     np.testing.assert_allclose(
-        zero, (G_sgd - G_sinkhorn).sum(1), atol=1e-02)  # cf convergence sgd
+        zero, (G_sgd - G_sinkhorn).sum(1), atol=1e-04)  # cf convergence sgd
```

Comment:
this test makes no sens since you should sum the absolute value to test if error and not just the difference:

```python
    np.testing.assert_allclose(G_sgd.sum(1),G_sinkhorn.sum(1), atol=1e-02)
````

same for sum(0)